pkg/volume/awsebs/aws_ebs.go

```diff
@@ -274,11 +291,11 @@ func (plugin *awsElasticBlockStorePlugin) ConstructVolumeSpec(volName, mountPath
 		names := strings.Split(volumeID, "/")
```

Comment:
The above line

```
if strings.HasPrefix(volumeID, "aws/") {
```

works only for filesystem, block symlinks have `aws:/` in their path.

Reply:
I see, because of makeGlobalPDPath... https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/awsebs/aws_ebs.go#L476 . :/

Reply:
So effectively there are 5 formats.

aws:/vol-abcd -> aws:///vol-abcd
aws/vol-abcd -> aws:///vol-abcd
aws:/us-east-1/vol-abcd -> aws://us-east-1/vol-abcd
aws/us-east-1/vol-abcd -> aws://us-east-1/vol-abcd
vol-abcd

formatVolumeID needs some tests.